cmd/checkpoint/main.go

```diff
@@ -115,36 +116,35 @@ func getPodsFromKubeletAPI() []byte {
 	return pods
 }
 
-func bothAPIServersRunning(pods v1.PodList) bool {
-	var kubeAPISeen, tempAPISeen bool
+func bothRunning(pods v1.PodList, an, tn, ns string) bool {
+	var actualPodSeen, tempPodSeen bool
 	for _, p := range pods.Items {
-		kubeAPISeen = kubeAPISeen || isKubeAPI(p)
-		tempAPISeen = tempAPISeen || isTempAPI(p)
-		if kubeAPISeen && tempAPISeen {
+		actualPodSeen = actualPodSeen || isPod(p, an, ns)
+		tempPodSeen = tempPodSeen || isPod(p, tn, ns)
+		if actualPodSeen && tempPodSeen {
 			return true
 		}
 	}
 	return false
 }
 
-func kubeSystemAPIServerRunning(pods v1.PodList, client clientset.Interface) bool {
+func actualPodRunning(pods v1.PodList, client clientset.Interface, n, ns string) bool {
```

Comment:
nit: Maybe would be better named generically `isPodRunning` or just `podRunning`?